tests/test_measures.py

```diff
@@ -89,7 +89,30 @@ def circuit():
     @pytest.mark.parametrize(
         "cases",
         [
+            [None, [0.9165164490394898, 0.0, 0.08348355096051052, 0.0]],
```

Comment:
Hi @mlxd, I thought I had fixed `qml.probs` errors from `test_measure.py` in https://github.com/PennyLaneAI/pennylane-lightning/pull/280. 

I am also a bit confused about the `qml.probs`. I thought it does not accept `wires=None` but supports `wires=[]` to obtain the probability of all wires, but it seems like `wires=None` also works for this test case....

Reply:
Yea, I thought so too. I think there may have been some additional changes in PL to cause this difference. Are you fine with the updates to the tests here, or do you think we should roll-back and opt for something different?

Reply:
Could this be related to https://github.com/PennyLaneAI/pennylane/pull/2432? Maybe it is a bug in PL?

Reply:
Aha, this was a test case that anyway should be failed. But I think it is anyway better to recover `wires=[]` case for the next test. I will just recover it.